package.json

```diff
@@ -12,7 +12,14 @@
     "lint": "eslint src/**/*.js demo/src/**/*.js",
     "size": "npm run build && size-limit",
     "test": "jest tests",
-    "build": "node ./build-packages.js",
+    "build": "npm-run-all --parallel build:*",
+    "build:submodule": "microbundle build --css-modules true --jsx React.createElement",
```

Comment:
I'm not the biggest fan of labeling this as "submodule" but it allows us to both build the main and use it as a reusable command for the rest so we're not repeating flags. Could separate it out into two commands, but thought I'd present this first.

Reply:
We can call it `build:dist`. IMO that makes it clearer that this command builds `/dist` folder.
What do you think?

Reply:
Fully agree, that's a better name.